DayTomato/DayTomato/Models/Pin.cs

```diff
@@ -38,15 +38,18 @@ public class Pin
 
 		public Pin() { }
 
-		public Pin(int type, string name, string description, int likes, double lat, double lng, string account)
+		public Pin(int type, string name, float rating, string description, int likes, double lat, double lng, string account, DateTime date)
 		{
 			_type = type;
 			_name = name;
+			_rating = rating;
 			_description = description;
 			_likes = likes;
 			_latitude = lat;
 			_longitude = lng;
 			_linkedAccount = account;
+			_reviews = new List<Review>();
+			_createDate = date;
 		}
 
 		public string Id { get { return _id; } set { _id = value; } }
```

Comment:
Is there a need for private fields? If not, it's possible to use the short initializer syntax to simplify things somewhat.
e.g.
```c#
public string Id { get; set; }
```
or
```c#
public string Id { get; private set; }
```